build.gradle

```diff
@@ -165,8 +165,12 @@ task printVersion {
 configure(subprojects.findAll {it.name != 'jaeger-thrift'}) {
    apply plugin: 'net.ltgt.errorprone'
    dependencies {
-     errorprone 'com.google.errorprone:error_prone_core:2.0.15'
+     errorprone 'com.google.errorprone:error_prone_core:2.1.1'
    }
+    tasks.withType(JavaCompile) {
+        // These are disabled because of a bug with errorprone. See https://github.com/google/error-prone/issues/750
```

Comment:
not following how that issue is related to disabling checks

Reply:
Enabling those checks causes a runtime exception in the error prone compiler

Reply:
so using `@Data` is a convenience, but turning off InstanceOfAndCastMatchWrongType sounds like it could mask real errors.

Reply:
`InstanceOfAndCastMatchWrongType` doesn't exist in the current version of errorprone that the project uses. 
Could you please justify your statement of `using @Data is a convenience`. Have there been no bugs made in manually writing out the convenience functions that `@Data/@Value` provides? If there have, is the cost of fixing the bugs introduced by incorrect hashcode/tostring/getters less than the potential time saved by having these new error checks enabled?
Has the time taken to write boilerplate code been factored into your argument?

Reply:
the methods that `@Data/@Value` generate can be just as easily created by the IDE.

Reply:
I really feel that you are arguing with me for the sake for arguing. 
How would you maintain consistency between the IDE generated code?

Should bumping a couple of versions have so much friction?